### PR TITLE
The KuCoin native package: implement open operations - part 1

### PR DESCRIPTION
**Pull request by skubarenko**
#### The first part:
- [x] [List exchange rate of coins](https://kucoinapidocs.docs.apiary.io/#reference/0/currencies-plugin/list-exchange-rate-of-coins(open))
- [x] [List languages](https://kucoinapidocs.docs.apiary.io/#reference/0/language/list-languages(open))
- [x] [Tick](https://kucoinapidocs.docs.apiary.io/#reference/0/market/tick(open))
- [x] [Order Books](https://kucoinapidocs.docs.apiary.io/#reference/0/market/order-books(open))
- [x] [Buy Order Books](https://kucoinapidocs.docs.apiary.io/#reference/0/market/buy-order-books(open))
- [x] [Sell Order Books](https://kucoinapidocs.docs.apiary.io/#reference/0/market/sell-order-books(open))

#### The second part (it will be in next merge request):
- [ ] [Recently deal orders](https://kucoinapidocs.docs.apiary.io/#reference/0/market/recently-deal-orders(open))
- [ ] [List trading markets](https://kucoinapidocs.docs.apiary.io/#reference/0/market/list-trading-markets(open))
- [ ] [List trading symbols tick](https://kucoinapidocs.docs.apiary.io/#reference/0/market/list-trading-symbols-tick-(open))
- [ ] [List trendings](https://kucoinapidocs.docs.apiary.io/#reference/0/market/list-trendings(open))
- [ ] [Get kline data](https://kucoinapidocs.docs.apiary.io/#reference/0/market/get-kline-data(open))
- [ ] [Get kline config](https://kucoinapidocs.docs.apiary.io/#reference/0/market/get-kline-config(open,-tradingview-version))
- [ ] [Get symbol tick](https://kucoinapidocs.docs.apiary.io/#reference/0/market/get-symbol-tick(open,-tradingview-version))
- [ ] [Get kline data](https://kucoinapidocs.docs.apiary.io/#reference/0/market/get-kline-data(open,-tradingview-version))
- [ ] [Get coin info](https://kucoinapidocs.docs.apiary.io/#reference/0/market/get-coin-info(open))
- [ ] [List coins](https://kucoinapidocs.docs.apiary.io/#reference/0/market/list-coins(open))

### CODE REVIEW

File: packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-constants.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
 const constants = {
     serverProductionUrl: 'https://api.kucoin.com',
+    listExchangeRateOfCoinsUri: '/v1/open/currencies',
+    listLanguages: '/v1/open/lang-list',
```
Comment (maxima-net) [inline packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-constants.ts]: is the 'uri' postfix skipped accidentally?

File: packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-rest-v1.ts [hunk 2]
```diff
@@ -107,6 +111,84 @@ export class KuCoinRestV1 {
...
+        group?: number,
+        limit?: number
+    }): Promise<KuCoinBuyOrderBook | KuCoinErrorResponseResult>;
+    async buyOrderBooks<T extends FieldsSelector<KuCoinBuyOrderBook>>(
+        parameters: {
+            symbol: CurrencyPair,
+            group?: number,
+            limit?: number
+        },
+        checkFields?: T
```
Comment (maxima-net) [inline packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-rest-v1.ts]: What if we make an interface 
```
interface buyOrderBooksParameters {
            symbol: CurrencyPair,
            group?: number,
            limit?: number
}
```
to reduce 'copy-paste' in overrides?

**Review by maxima-net — CHANGES_REQUESTED** [review]

File: packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-constants.ts (hunk 1 above)
Comment (skubarenko) [reply packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-constants.ts]: My mistake. Ok, I'll fix it.

File: packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-rest-v1.ts (hunk 2 above)
Comment (skubarenko) [reply packages/crypto-kraken-kucoin-api/src/kucoin-rest-v1/kucoin-rest-v1.ts]: Ok, I agree with you

**Review by maxima-net — APPROVED** [review]

**Pull request merged**